src/pages/home/report/ReportActionsView.js

```diff
@@ -26,6 +26,9 @@ import Visibility from '../../../libs/Visibility';
 import Timing from '../../../libs/actions/Timing';
 import CONST from '../../../CONST';
 import themeColors from '../../../styles/themes/default';
+import Navigation from '../../../libs/Navigation/Navigation';
+import compose from '../../../libs/compose';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
```

Comment:
I'm assuming that `withWindowDimensions` is required in order to enable `isSmallScreenWidth`. is that correct?

Reply:
yeah